scripts/run_OQStandard.sh

```diff
@@ -147,61 +155,62 @@ echo "All initialization files exist - You're ready to go."
 #exit 0
 #fi
 
+oq dbserver start || true
```

Comment:
Starting dbserver manually here (instead of waiting till the first run of `oq engine run` to start dbserver) is a workaround to prevent `tee` in the first call of `oq engine --run ... 2>&1 | tee ...` from getting stuck.

I changed `&> logfile` to `2>&1 | tee logfile` in various places throughout the script because I wanted to not only save the logs but also see OpenQuake output at the same time.  Sorry, I neglected to document that in the commit log.  😅